camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -274,7 +274,6 @@ public void testJobFailDueToOffsetRangeCallError() throws Exception {
     setupJobFailDueToOffsetRangeCallError();
     job = new CamusJob(props);
     job.run(EtlInputFormatForUnitTest.class, EtlMultiOutputFormat.class);
-    verifyJobSucceed();
```

Comment:
This job is supposed to fail, so should not verify job succeed. 
This somehow passes on maven, but fails on ligradle.